packages/fluentui/react-northstar/src/components/Dialog/utils/index.tsx

```diff
@@ -15,3 +15,26 @@ export function disableBodyScroll(target: HTMLElement): void {
     target.addEventListener('touchmove', _disableIosBodyScroll, { passive: false, capture: false });
   }
 }
+
+const dialogsCounterAttribute = 'fluent-dialogs-count';
+
+export const lockBodyScroll = target => {
```

Comment:
```suggestion
export const lockBodyScroll = (target: Document) => {
```

I suggest to keep`target` as `document` and type it

Reply:
Alright, no problem. Gonna change, just out of curiosity.. Why?

Reply:
https://github.com/microsoft/fluentui/blob/4eb6f33eeb419a3189b8c7d2c5e5675fd86a77fd/packages/utilities/src/scroll.ts#L105

1. I would like to converge these functions
2. `body` may not exist on `document` and be `null`: https://developer.mozilla.org/en-US/docs/Web/API/Document/body

Reply:
Done.